tests/kola-ci-self/tests/kola/autopkgtest-reboot

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copy of the reboot example from https://salsa.debian.org/ci-team/autopkgtest/raw/master/doc/README.package-tests.rst
+set -xeuo pipefail
+case "${AUTOPKGTEST_REBOOT_MARK:-}" in
```

Comment:
Bikeshed: WDYT about just `KOLA_REBOOT_MARK`? It's good to know that the concept comes from Debian's autopkgtest, though it seems cleaner to have all the env vars exposed consistently start with `KOLA_`.

Reply:
Similarly, the variable is part of the spec.  I think using *exactly* the API from autopkgtest is valuable because it helps us more easily consume existing test suites.

Reply:
Ack, sure WFM.